CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.11.0
+
+* Breaking change: Prefix generated Dart proto imports by proto file path instead of by package.
```

Comment:
I don't think this change is actually breaking.
The generated code should still work with the older protobuf.